.github/workflows/version.yml

```diff
@@ -124,7 +124,8 @@ jobs:
   upload:
     name: Upload
     needs: ["signing", "prepare"]
-    if: github.event_name == 'push' || github.event_name == 'workflow_dispatch'
+    # Make sure to run this job even if signing has been skipped.
+    if: ${{ !(failure() || cancelled()) && (github.event_name == 'push' || github.event_name == 'workflow_dispatch') }}
```

Comment:
Unfortunately a bit hacky :cry: . But currently there seems to be no nicer way.

I've got the syntax from this StackOverflow discussion: https://stackoverflow.com/questions/69354003/github-action-job-fire-when-previous-job-skipped#comment136588503_77066140